f3.go

```diff
@@ -410,9 +403,15 @@ func (m *F3) resumeInternal(ctx context.Context) error {
 		return err
 	}
 
+	walPath := filepath.Join(m.diskPath, "wal", strings.ReplaceAll(string(m.manifest.NetworkName), "/", "-"))
```

Comment:
not directly related, but the names `.` and `..` will cause trouble, even without slashes. Although I guess that should just fail?